### PR TITLE
Add emojis to command output, edit UG

### PR DESCRIPTION
**Pull request by ruoyann**
image

Problems: 
- When changing to dark theme, emojis can appear weird as shown below: 
image
- Emojis slightly cut off especially for the clock emoji

Improvements
- Studied hours still in text form. Any emojis we can use to replace that? 
image

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -76,9 +76,10 @@ You're on the right track to productivity!
 
 1. Refer to the [Features](#features) section for more commands and features in StudyTracker.
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-Check out **<a href="#command-structure">How to read Commands in StudyTracker</a>** section if you are unfamiliar with CLI syntax!
-</div>    
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:** 
+Check out **<a href="#command-structure">How to read Commands in StudyTracker</a>** section if you are unfamiliar 
+with CLI syntax!
+</div>
```
Comment (ruoyann) [inline docs/UserGuide.md:82]: Sorry idk why I changed this line

File: docs/UserGuide.md [hunk 2]
```diff
@@ -229,20 +231,20 @@ Scroll down the list to view all the study spots!
 * `Name` — name of study spot
 * `Tags` — user-defined tags for the study spot
 * `Address` — address of study spot
-* `Operating Hours` — operating hours of study spot, weekdays and weekends
-* `Favourite` — this icon shows if the study spot is added to favourites
+* `Operating Hours` — operating hours of study spot on weekdays and weekends respectively
+* `Favourite` — the presence of this icon shows that the study spot is added to favourites, and vice versa
```
Comment (mslevis) [inline docs/UserGuide.md:235]:
```suggestion
* `Favourite` — this icon indicates the study spot is a favourite
```
Would this be more straightforward?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -706,7 +708,7 @@ Alias is the command for you. The `alias` command can help you to shorten input
 
 | Input     | Output | Usage of Alias | Output |
 | ----------- | ----------- | ---------- | ------------- |
-| `alias al/MyList cmd/list` | `Added alias myList: [list]` | `myList` | `Listed all study spots` |
+| `alias al/MyList cmd/list` | Added alias myList: [list] | `myList` | `Listed all study spots` |
```
Comment (mslevis) [inline docs/UserGuide.md:711]: * Is there a reason why `Added alias myList: [list]` should not be in codeblocks?
* `myList` should be `MyList` in the output column
* this example may not be the best as we wrote that `alias` command helps to shorten commands but it lengthened the command input in this example...
* Perhaps we should include that alias is case-sensitive?

**Review by mslevis — CHANGES_REQUESTED** [review]
Nice! Just some minor changes.

File: src/main/java/seedu/address/model/studyspot/StudySpot.java [hunk 4]
```diff
@@ -156,43 +156,50 @@ public String toString() {
...
         if (!amenities.isEmpty()) {
-            builder.append("; Amenities: ");
             amenities.forEach(builder::append);
         }
 
+        if (getStudiedHours().getHours() > 0) {
+            builder
+                    .append("(")
+                    .append(getStudiedHours())
+                    .append(" hours studied)")
```
Comment (mslevis) [inline src/main/java/seedu/address/model/studyspot/StudySpot.java:199]: Can cater for `1 hour studied` as well

File: docs/UserGuide.md (hunk 3 above)
Comment (ruoyann) [reply docs/UserGuide.md:711]: 1. Isn't Added alias myList: [list] an output and not a command? I think for the UG all the outputs have no codeblocks. 
2. okay!
3. changed MyList to ml
4. okay!

File: src/main/java/seedu/address/model/studyspot/StudySpot.java (hunk 4 above)
Comment (ruoyann) [reply src/main/java/seedu/address/model/studyspot/StudySpot.java:199]: ok thank u

**Review by mslevis — APPROVED** [review]

**Pull request merged**